BinaryEncoding.md

```diff
@@ -119,11 +119,15 @@ The type section declares all function signatures that will be used in the modul
 | entries | `type_entry*` | repeated type entries as described below |
 
 #### Type entry
-| Field | Type | Description |
+| Field | Type/Value | Description |
 | ----- |  ----- | ----- |
+| constructor | `0x05` | the function type constructor |
```

Comment:
On Tue, Apr 5, 2016 at 6:53 PM, Luke Wagner <notifications@github.com>
wrote:

> In BinaryEncoding.md
> <https://github.com/WebAssembly/design/pull/636#discussion_r58574302>:
>
> >  | ----- |  ----- | ----- |
> > +| constructor | `0x05` | the function type constructor |
>
> And now I remember the original question that motivated my post: given
> this single index space, then why does the constructor field need to
> start at any other value than 0? Function (and later struct, array)
> constructors are not in the same index space anymore.
>
Yep, good point.



> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/WebAssembly/design/pull/636/files/bb8a6e43af9bc6b5b904a5124062b59ae06d1304#r58574302>
>